manifests/sysctl.pp

```diff
@@ -0,0 +1,250 @@
+# Sets sysctl settings that are useful from a general 'modern system'
+# point of view.
+#
+# There are also items in this list that are particularly useful for
+# general system security.
+#
+# See the kernel documentation for the functionality of each variable.
+#
+# @params core_dumps Boolean
+#   If true, enable core dumps on the system.
+#
+#   As set, meets CCE-27033-0
+#
+class simp::sysctl (
+  # Performance Related Settings
+  $net__netfilter__nf_conntrack_max      = '655360',
+  $net__unix__max_dgram_qlen             = '50',
```

Comment:
Bake these as Stdlib::Compat::Integer

You can fix some with Enum and others will need a Pattern since they have spaces